torch/nn/attention/_flex_attention.py

```diff
@@ -385,15 +526,18 @@ def create_block_mask(
         block_mask (tuple): A tuple of (kv_num_blocks, kv_indices, q_num_blocks, q_indices,
                             KV_BLOCK_SIZE, Q_BLOCK_SIZE) which represents the block mask.
     """
+    is_score_mod = len(inspect.signature(fn).parameters) == len(
```

Comment:
Unfortunately I don't think this works: 
<img width="454" alt="image" src="https://github.com/pytorch/pytorch/assets/6355099/4014b287-1a6c-45c6-af40-eeec5a923ac3">